integration/client/container_linux_test.go

```diff
@@ -1417,3 +1419,55 @@ func TestShimOOMScore(t *testing.T) {
 	case <-statusC:
 	}
 }
+
+// TestIssue9103 is used as regression case for issue 9103.
+//
+// The runc-fp will kill the init process so that the shim should return stopped
+// status after container.NewTask. It's used to simulate that the runc-init
+// might be killed by oom-kill.
+func TestIssue9103(t *testing.T) {
+	if os.Getenv("RUNC_FLAVOR") == "crun" {
+		t.Skip("skip it when using crun")
+	}
+
+	t.Skip("Wait for #9104")
```

Comment:
Maybe take this out so we can see it fail, and then rebase after #9104?

Reply:
Good point. Updated!